libs/designsystem/src/lib/components/charts/shared/chart-js-service/chartjs-plugin-marker/chartjs-plugin-marker.ts

```diff
@@ -7,7 +7,7 @@
 import { Chart } from 'chart.js';
 import { valueOrDefault } from 'chart.js/helpers';
 
-import { ChartDataset } from '../../chart.types';
+import { ChartDataset } from '../../../';
```

Comment:
Can this cause problems down the line? Historically we have had some issues referencing _components_ via barrel files: https://github.com/kirbydesign/designsystem/issues/1467

Reply:
I cannot see why - especially since no components are being introduced via barrel files in this plugin. I've made sure to not introduce any circular dependencies (`npm run build` complained about them when i've accidentally introduced them).